sdk/keyvault/azure-keyvault-keys/tests/test_keys_async.py

```diff
@@ -215,18 +222,45 @@ async def test_key_crud_operations(self, client, **kwargs):
         self.assertIsNotNone(deleted_key)
         self.assertEqual(created_rsa_key.id, deleted_key.id)
 
-    @ResourceGroupPreparer(random_name_enabled=True)
     @KeyVaultPreparer()
-    @KeyVaultClientPreparer()
-    async def test_key_list(self, client, **kwargs):
+    async def test_backup_restore(self, azure_keyvault_url, **kwargs):
+        client = self.create_client(azure_keyvault_url)
+        self.assertIsNotNone(client)
+
+        key_name = self.get_resource_name("keybak")
+        key_type = "RSA"
+
+        # create key
+        created_bundle = await client.create_key(key_name, key_type)
+        self.assertEqual(key_type, created_bundle.key_type)
+
+        # backup key
+        key_backup = await client.backup_key(created_bundle.name)
+        self.assertIsNotNone(key_backup, "key_backup")
+
+        # delete key
+        await client.delete_key(created_bundle.name)
+        # can add test case to see if we do get_deleted should return error
```

Comment:
🤔

Reply:
🤔, indeed

I only moved this test case up to match the ordering of the synchronous tests, so I don't think I wrote this line... in any case, `test_purge` covers what I think is being described here